### ISSUE TITLE
Bernoulli takes `logit_p` as argument, but it's not documented

### ISSUE THREAD

**Issue by MarcoGorelli** [pymc-devs/pymc#5006]
It's possible to specify `Bernoulli` with `logit_p` instead of `p`, but it's not documented:

https://github.com/pymc-devs/pymc3/blob/59ce9af97f0b840f992322f33ec8dee203d7d1c0/pymc3/distributions/discrete.py#L327-L342

Pull requests welcome!

**Comment by farhanreynaldo** [comment]
Hi, would it be sufficient if I state the `logit_p` as follows:

```
Parameters 
     ---------- 
     p: float 
         Probability of success (0 < p < 1). 
     logit_p: float 
         Log odds for the probability of success. 
     """
 ```

**Comment by MarcoGorelli** [comment]
Looks good to me

**Comment by ricardoV94** [comment]
We usually have a text in the docstrings mentioning the mathematical relationship between the alternative parametrizations, and also mention in the `Parameters block` when one is an alternative to another.

**Comment by farhanreynaldo** [comment]
$
\mu &= \frac{n(1-p)}{p} \\
\alpha &= n

**Comment by farhanreynaldo** [comment]
Do you mean something like this?

```
    The bernoulli distribution can be parametrized either in terms of p or logit_p.
    The link between the parametrizations is given by

    .. math::
        logit(p) &= ln\left (\frac{p}{1-p} \right)

    Parameters
    ----------
    p: float
        Probability of success (0 < p < 1).
    logit_p: float
        Alternative log odds for the probability of success.
```

**Comment by ricardoV94** [comment]
That looks perfect

**Comment by MarcoGorelli** [comment]
Closed in #5037
